cmd/flags_node.go

```diff
@@ -9,34 +9,56 @@ import (
 	flag "github.com/spf13/pflag"
 
 	"github.com/celestiaorg/celestia-node/node"
+	"github.com/celestiaorg/celestia-node/params"
 )
 
 var (
-	nodeStoreFlag  = "node.store"
-	nodeConfigFlag = "node.config"
+	nodeStoreFlag   = "node.store"
+	nodeConfigFlag  = "node.config"
+	nodeNetworkFlag = "node.network"
 )
 
 // NodeFlags gives a set of hardcoded Node package flags.
-func NodeFlags(tp node.Type) *flag.FlagSet {
+func NodeFlags() *flag.FlagSet {
 	flags := &flag.FlagSet{}
 
 	flags.String(
 		nodeStoreFlag,
-		fmt.Sprintf("~/.celestia-%s", strings.ToLower(tp.String())),
+		"",
 		"The path to root/home directory of your Celestia Node Store",
 	)
 	flags.String(
 		nodeConfigFlag,
 		"",
 		"Path to a customized node config TOML file",
 	)
+	flags.String(
+		nodeNetworkFlag,
+		"",
```

Comment:
nit: We can set `params.DefaultNetwork()` here so that when we `ParseNodeFlags`, the `network` is never empty, and this reduces code branchiness.

Reply:
The main reason for doing this is so that it is clear when a user overrides it: This is helpful because when it is done explicitly, we definitely need to call SetDefaultNetwork, because the params will not hear about it otherwise.

Reply:
For this reason it will not reduce code branchiness: We will have to test if it is custom anyways, and then have some weird/complex logic to determine if it was the default value or not